### PR TITLE
Install transifex

### PR DESCRIPTION
**Pull request by Bluesmile82**
Install transifex
---

Install transifex, add language selector, start adding translation components

- Transifex has been installed with a temporary account of Conservation international. 

- A new language selector has been added to the page on the right of the navbar.

- We have 6 fixed languages to translate.

- Currently only the string Journey from the navbar is translated to spanish to test.
- The rest of the strings marked as translatable should show differently on languages different than english (ȦȦɃǾǾŬŬŦ)  as we are using PseudoTranslationPolicy
- Only part of the UI is translated for now. Some missing pieces from the map and logged in pages have to be marked as translatable. This will be done on a different PR.

## Testing instructions

Change the language on the Language selector and check that you see the translation of journeys in spanish and the strings showing differently on the rest of strings. 

## Tracking

https://vizzuality.atlassian.net/browse/RA2-95?atlOrigin=eyJpIjoiZGNhYzk2YTE1Y2IzNGZlNTkzN2FkZjYyY2QzYzJlYmIiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by Bluesmile82** [comment]
False positive on firefox but Ready to review!

**Comment by davidsingal** [comment]
Screenshot 2023-03-30 at 13 34 27
I miss a shadow or something like that, but more important I'd prefer to show the language names instead of the code in the dropdown.

**Comment by Bluesmile82** [comment]
@davidsingal  Ready again. The scripts were not working correctly and I had to copy the env variables so we now have NEXT_PUBLIC_TRANSIFEX_TOKEN and TRANSIFEX_TOKEN. I know is not ideal but  I couln't make it work with the --token and --secret of transifex-cli.

File: frontend/README.md [hunk 1]
```diff
@@ -49,6 +53,49 @@ Every time we update the environment variables, we need to update them in the fo
 - Update the environment variables in the `README.md` file.
 - Finally, you have to update the environment variables in the `.env.example` file.
 
+## Translation
+
+The aplication is translated with the [transifex native](https://www.transifex.com/native/) service.
```
Comment (clementprdhomme) [inline frontend/README.md:58]:
```suggestion
The application is translated with the [Transifex Native](https://www.transifex.com/native/) service.
```

File: frontend/README.md [hunk 2]
```diff
@@ -49,6 +53,49 @@ Every time we update the environment variables, we need to update them in the fo
...
+
+## Selecting the translation locale
+
+There is a language switcher component available on the menu.The locale is then retrieved on the App component to set the translation.
+
+## To translate strings
+
+The translation of strings will depend where is the string:
+
+### Strings inside Components
```
Comment (clementprdhomme) [inline frontend/README.md:87]:
```suggestion
#### Strings inside Components
```

**Review by clementprdhomme — COMMENTED** [review]
I only did a superficial review.

File: frontend/.env.example [hunk 3]
```diff
@@ -1,3 +1,7 @@
 NEXT_PUBLIC_API_HOST='https://www.domain.org'
 NEXT_PUBLIC_GOOGLE_ANALYTICS='UA-XXXXX-X'
 NEXT_PUBLIC_STATIC_JOURNEYS=true
+NEXT_PUBLIC_TRANSIFEX_TOKEN=
+NEXT_PUBLIC_TRANSIFEX_SECRET=
+TRANSIFEX_TOKEN=
+TRANSIFEX_SECRET=
```
[the remark this answers is not in the document]
Comment (clementprdhomme) [reply frontend/.env.example:7]: @davidsingal Maybe you can have a look at this? Let me know if you can't.

[the remark this answers is not in the document]
Comment (davidsingal) [reply frontend/.env.example:7]: I'll do that.`NEXT_PUBLIC_TRANSIFEX_TOKEN ` it's not needed both, `NEXT_PUBLIC` works on the server and client side.

**Comment by Bluesmile82** [comment]
Is this ready to merge? @davidsingal thanks for the help

**Pull request merged**